### DISCUSSION TITLE
Mod minimum reserved amount for synthesis

### DISCUSSION THREAD

**Discussion by Alinkk** [Lailloken/Exile-UI#26] [Ideas & Feedback]
Mod minimum reserved amount for synthesis(maybe customize cause _**the priority system currently only calculates the route to complete one set of the priority list**_ ) 
excess amount mods are used for burn suggestions 

I can't express it clearly, hope you can understand

**Comment by Alinkk** [comment]
Found that it has been discussed in issues

**Comment by Lailloken** [comment]
Just for clarification:
- do you want prio-surplus to be highlighted in burn lists?
- or is there a problem at the moment that causes prio-surplus to appear in the burn lists?

**Reply by Alinkk** [reply]
prio-surplus to be highlighted in burn lists

**Reply by Lailloken** [reply]
Alright, I'll look into it. Thanks for your feedback.
